app/src/app/[lng]/[inventory]/preferences/activities/ACTIVITY_ITEMS.tsx

```diff
@@ -0,0 +1,26 @@
+export const ACTIVITY_ITEMS = [
+  {
+    id: "commercial-and-institutional",
+    icon: "/assets/preferences/waste/commercial-institutional.svg",
+  },
+  {
+    id: "manufacturing-industries",
+    icon: "/assets/preferences/waste/manufacturing.svg",
```

Comment:
Do we need the whole path for all of these or would it be sufficient to have just the base name (e.g. `"manufacturing.svg"` or `"waste/manufacturing.svg"`) here? The /assets/preferences/waste prefix can be added in the place this is used imo.